src/components/Datepicker/Datepicker.vue

```diff
@@ -99,6 +102,7 @@ import { ClickOutside, Tooltip } from '../../directives'
 import { includes } from '../../utils'
 import { SbTextField } from '../TextField'
 import { SbPopover } from '../Popover'
+import { SbButton } from '../Button'
```

Comment:
can we change to ts on the `script`?

Reply:
For this i will change in a future task, because for this PR i tried to work only inside the `DatepickerTime` component